🔧 chore: update asset paths and local preview instructions

faizanmsiddiqui opened the pull request:
## 🎯 Purpose

Update asset paths in `index.html` to use absolute references for improved reliability. Revise local preview instructions in the README to recommend running a local server instead of opening `src/index.html` directly.

## 💡 Notes

No functional changes were made to the application.

faizanmsiddiqui committed:
🔧 chore(html): update asset paths in index.html

Modified asset paths in `index.html` to use absolute references for
the favicon, CSS, and JavaScript files. This change ensures that the
assets are correctly loaded regardless of the current directory,
improving the reliability of the page rendering.

No functional changes were made to the application.

faizanmsiddiqui committed:
📝 docs: update local preview instructions in README

Revised the instructions for previewing the template locally. The previous
method of opening `src/index.html` directly in the browser has been replaced
with guidance on running a local server using Node.js or Python. This change
aims to provide a more accurate and effective way for users to preview the
page, ensuring they can see the full functionality of the template.

The updated instructions include commands for both Node.js and Python, along
with the URL to access the local server.

imfsiddiqui commented on src/index.html:
```diff
@@ -13,10 +13,10 @@
-    <link rel="icon" href="./assets/icons/favicon.svg" type="image/svg+xml" />
+    <link rel="icon" href="/assets/icons/favicon.svg" type="image/svg+xml" />
     <link rel="preconnect" href="https://cdn.jsdelivr.net" crossorigin />
-    <link rel="stylesheet" href="./assets/css/main.css" />
-    <script type="module" src="./assets/js/main.js"></script>
+    <link rel="stylesheet" href="/assets/css/main.css" />
+    <script type="module" src="/assets/js/main.js"></script>
```
Instructions added in the README.md for it.

faizanmsiddiqui committed:
📝 docs: update local preview instructions in index.md

Revised the instructions for previewing the template locally. The previous
method of opening `src/index.html` directly in the browser has been replaced
with guidance on running a local server from the `src/` directory. This change
includes examples using Node.js and Python to enhance usability and ensure
users can easily view the page at `http://localhost:5000`.

imfsiddiqui commented on src/index.html:
```diff
@@ -13,10 +13,10 @@
     />
     <meta name="author" content="Faizan Siddiqui" />
-    <link rel="icon" href="./assets/icons/favicon.svg" type="image/svg+xml" />
+    <link rel="icon" href="/assets/icons/favicon.svg" type="image/svg+xml" />
```
Instructions already added in the README.md.
